rules/okta_rules/okta_new_behavior_accessing_admin_console.py

```diff
@@ -0,0 +1,41 @@
+from panther_base_helpers import deep_get, get_val_from_list, okta_alert_context
+
+
+def rule(event):
+    if event.get("eventtype") != "policy.evaluate_sign_on":
+        return False
+
+    target_app_names = get_val_from_list(
+        event.get("target", [{}]), "displayName", "type", "AppInstance"
+    )
+
+    if "Okta Admin Console" not in target_app_names:
+        return False
+
+    behaviors = deep_get(event, "debugContext", "debugData", "behaviors")
+    if behaviors:
+        return "New Device=POSITIVE" in behaviors and "New IP=POSITIVE" in behaviors
+
+    return (
+        deep_get(
+            event, "debugContext", "debugData", "logOnlySecurityData", "behaviors", "New Device"
+        )
+        == "POSITIVE"
+        and deep_get(
+            event, "debugContext", "debugData", "logOnlySecurityData", "behaviors", "New IP"
+        )
+        == "POSITIVE"
+    )
+
+
+def title(event):
+    return (
```

Comment:
Same comment re: default.

Reply:
default strings in human readable titles makes sense, ty